website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -3,102 +3,57 @@
 
 ### When not to use
 
-- To display persistent messages or promotional content, consider [Alert](/components/alert).
+- To display contextual associative errors (e.g. form validation), or promotional content, consider [Alert](/components/alert).
 - As a dialog to confirm an action, consider [Modal](/components/modal).
 
 ## Color
 
-<Doc::Layout @spacing="12px">
-  <Hds::Toast @onDismiss={{this.noop}} as |T|>
-    <T.Title>Neutral toast title</T.Title>
-    <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="highlight" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Highlight toast title</T.Title>
-    <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Success toast title</T.Title>
-    <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="warning" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Warning toast title</T.Title>
-    <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="critical" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Critical toast title</T.Title>
-    <T.Description>Lorem ipsum dolar sit amet, consecteu adipiscig elit null dignissim felis.</T.Description>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
-
-Use color logically.
-
-- **Neutral** to provide general information to the user about an ongoing process.
-- **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
-- **Success** to indicate a successful action was completed.
-- **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
-
-- **Critical** to indicate error or critical issues resulting from a failed action.
+![A list different types of toasts, starting from neutral all the way to critical](/assets/components/toast/toast-colors.png)
```

Comment:
I'd lean towards images... but on the condition that they have good alt text 😆 

Reason being, we had to swap out the real examples for modals/flyouts for images because including the real components caused a11y tests to fail. 

One downside tho is if we ever change the design we have to make new images. 


Side note: I think we can improve the alt text here
```suggestion
![The color argument changes the background and title color: neutral is grey, highlight is purple, success is green, warning is orange, critical is red.](/assets/components/toast/toast-colors.png)
```